src/bin-utils/index.test.js

```diff
@@ -305,6 +311,69 @@ test(
   },
 )
 
+test('specificHelpScript: returns no script available', () => {
+  const config = {scripts: {}}
+  const message = specificHelpScript(config, 'foo')
+  const expected = chalk.yellow('Script foo does not exist')
+  expect(message).toBe(expected)
+})
+
+test(
+  'specificHelpScript: do not display script with flag hiddenFromHelp=true',
+  () => {
+    const config = {
+      scripts: {
+        foo: {
+          description: 'the foo script',
+          script: 'echo "foo"',
+          hiddenFromHelp: true,
+        },
+      },
+    }
+    const message = specificHelpScript(config, 'foo')
+    const expected = chalk.yellow('Script foo does not exist')
+    expect(message).toBe(expected)
+  },
+)
+
+test('specificHelpScript: formats a nice message', () => {
+  const config = {
```

Comment:
Alright, when I use prefix-matches, I will be getting a nested object as the first member of the resultant Array. And the script can be a string, number or object. How would I validate it and convert it to 
```
{
    name ,
    description,
    script,
 }
```
format, so I can stringify it ?